### ISSUE TITLE
Doesn't work on Redmi Note 12S

### ISSUE THREAD

**Issue by anyuta1166** [tehcneko/XiaomiEsimLPA#5]
Redmi Note 12S, Android 14 with HyperOS, removable eSIM (5ber).
Can't turn on "Use eSIM" option - it throws an error "couldn't activate eSIM".
Tried 5ber in SIM slot 1 and slot 2 - no difference.
Both 5ber application and OpenEUICC works fine.

**Comment by anyuta1166** [comment]
All the phones listed in "tested devices" are based on Qualcomm Snapdragon. Mine device is based on MediaTek Helio.
Maybe this module works only on Qualcomm?

**Comment by alyx404** [comment]
> All the phones listed in "tested devices" are based on Qualcomm Snapdragon. My device is based on MediaTek Helio.

Yeah....It seems like listed devices are manually tested so only 4 devices.
My K60Pro(Socrates) works fine tho.

> Maybe this module works only on Qualcomm?

You should check LPA App's version (latest is 1.5.7) and [Settings - My device - Detailed info and specs - Status] and see if there's EID number. No EID = No luck.
If you have EID Listed but meet problems, update LPA app in apkmirror then. Version 1.5.5 crashed when clicking Manage eSIM on my K60Pro.
